core/src/kvs/ds.rs

```diff
@@ -483,7 +483,7 @@ impl Datastore {
 				// Display information in the logs
 				info!("Credentials were provided, and no root users were found. The root user '{}' will be created", username);
 				// Create and save a new root users
-				let stm = DefineUserStatement::from((Base::Root, username, password));
+				let stm = DefineUserStatement::from((Base::Root, username, password, "owner"));
```

Comment:
nit: this should be a static, so we aren't guessing what the value should be

Reply:
Thanks @phughk! Done in [ec51ca2](https://github.com/surrealdb/surrealdb/pull/4116/commits/ec51ca28269e3feca1780505fc8c6447174003ec).